make/go/dep_protoc.mk

```diff
@@ -10,8 +10,8 @@ $(call _assert_var,CACHE_INCLUDE)
 $(call _assert_var,CACHE_BIN)
 
 # Settable
-# https://github.com/protocolbuffers/protobuf/releases 20220929 checked 20220930
-PROTOC_VERSION ?= 21.7
+# https://github.com/protocolbuffers/protobuf/releases 20221214 checked 20230104
+PROTOC_VERSION ?= 21.12
```

Comment:
I wonder if we can remove this already? I don't think it is used anymore in oss or internally.

Reply:
I think we still use it in buf for testing IIRC. I think we could remove it from other projects if we wanted.